addons/mrp/models/mrp_production.py

```diff
@@ -686,6 +686,9 @@ def _get_ready_to_produce_state(self):
     def action_confirm(self):
         self._check_company()
         for production in self:
+            # Avoid confirming it twice
+            if production.state == 'confirmed':
```

Comment:
!= 'draft' instead. So we would avoid the same issue if he closes the WO or something else

Reply:
Alright, done 😁